pennylane/tape/tape.py

```diff
@@ -273,14 +306,6 @@ def __repr__(self):
     def __enter__(self):
         QuantumTape._lock.acquire()
         try:
-            if not QueuingContext.recording():
-                # if the tape is the first active queuing context
-                # monkeypatch the operations to support the new queuing context
-                with contextlib.ExitStack() as stack:
-                    for mock in mock_operations():
-                        stack.enter_context(mock)
-                    self._stack = stack.pop_all()
-
```

Comment:
How come this was removed?

Reply:
This is the monkeypatching that helped support both the old and new core at the same time :) Now that the old core is gone, they can be removed